shoes-swt/lib/shoes/swt/list_box.rb

```diff
@@ -28,7 +28,7 @@ def initialize(dsl, parent)
       end
 
       def update_items
-        @real.items = @dsl.items.map(&:to_s)
+        @real.items = @dsl.items.to_a
```

Comment:
Looks to me like Shoes 3 did coercion to string, so probably best keep it for compatibility. I can imagine someone passing things that aren't strings along (in particular primitives like numbers), and it'd be frustrating if it didn't do the "obvious" thing.